department-of-veterans-affairs/caseflow
# Backend cleanup

Resolves #8138 

### Description
This PR renames room_info to room so that the fields match between hearings and hearing_days. It also updates children records when the parent record is updated.

### Testing Plan
1. Update a parent record that contains children records
1. Confirm the children records are updated appropriately
